package.json

```diff
@@ -18,6 +19,7 @@
     "formik": "^2.2.0",
     "html-loader": "^1.3.1",
     "html-webpack-plugin": "^4.5.0",
+    "moment": "^2.29.1",
```

Comment:
Can we not use `moment`? If it's extremely necessary for it's working, please move it to `peerDependency`. @edwinbbu _a

Reply:
@vinay0x _a I have moved moment to peerDependency

Date input works without moment but I was not able to format the date properly to the way we required. Javascript Date only provides few format options. 

<img width="680" alt="Screenshot 2020-10-27 at 9 35 46 AM" src="https://user-images.githubusercontent.com/17916573/97256229-7e3b9000-1838-11eb-9646-6ed44f2d66bd.png">